gpu_bdb/queries/q22/gpu_bdb_query_22.py

```diff
@@ -69,7 +68,6 @@ def main(client, config):
 
     output_table = output_table[keep_columns]
 
-    date_dim = date_dim.map_partitions(convert_datestring_to_days)
```

Comment:
Had to remove it here as the PR added the code to the utils. Adding twice made the code to give zero values as result. 

https://github.com/VibhuJawa/gpu-bdb/blob/1dcc3f5e94ca8d8ff4f633d90b45371e6f01db23/gpu_bdb/bdb_tools/q22_utils.py#L47